### ISSUE TITLE
Inline color picker slider sometimes displays the wrong cursor and cannot be dragged for #FF0000 but works for #00FF00

### ISSUE THREAD

**Issue by ccammack** [cocopon/tweakpane#630]
Here's an example using 4.0.3:

[https://codepen.io/ccammack/pen/WNaogKa?editors=1111](https://codepen.io/ccammack/pen/WNaogKa?editors=1111)

![2024-10-26 21_11_09-Tweakpane inline picker bug - Brave](https://github.com/user-attachments/assets/e683a9eb-b811-4dd4-a549-7de0dc2df569)

**Comment by cocopon** [comment]
Please write out the steps to reproduce and the specific issue you're seeing. A screen capture would also be helpful.

**Comment by ccammack** [comment]
1. Using the following code or the sandbox link provided, open one of the red color pickers (#ff0000)
2. Note that the arrow cursor does not reliably change to a hand while hovered, as shown in the provided image, and that the slider cannot be dragged while the wrong cursor is shown
3. Open the green picker  (#00ff00) and note that it works properly

```javascript
import * as tweakpane from "https://cdn.skypack.dev/tweakpane@4.0.3";

const pane = new tweakpane.Pane({ });

const PARAMS_0 = { color: '#ff0000', };
pane.addBinding(PARAMS_0, 'color', { picker: 'inline', expanded: false, });

const PARAMS_1 = { color: '#00ff00', };
pane.addBinding(PARAMS_1, 'color', { picker: 'inline', expanded: false, });

const PARAMS_2 = { color: '#ff0000',};
pane.addBinding(PARAMS_2, 'color', { picker: 'inline', expanded: false, });
```

**Comment by cocopon** [comment]
The sandbox that you shared seems to work fine in my environment.

- macOS 10.13.6.7
- Google Chrome 130.0.6723.70 (latest)

https://github.com/user-attachments/assets/ce51003a-5735-4eb6-aec2-1ccd663d4019

**Comment by cocopon** [comment]
Ah, I understand what you mean now.

The knob itself doesn’t have a hit area, but the track does, in the current implementation. This can be improved.

**Comment by cocopon** [comment]
Fixed in 4.0.5.
